client/src/components/Footer/Footer.css

```diff
@@ -0,0 +1,47 @@
+@import url('https://fonts.googleapis.com/css?family=Reem+Kufi&display=swap'); /* Will eventually delete */
```

Comment:
This should be set globally, in `App.css`

Reply:
Updated the component CSS to reflect this.